data/scrape/README.md

```diff
@@ -1,3 +1,13 @@
+* Languages: 251
+  * Broad transcription files: 259
+  * Narrow transcription files: 129
+* Languages & dialects: 15
```

Comment:
This isn't really "languages & dialects" since it's a smaller number than languages. You can either rename it "dialects" (I think that would be consistent with "scripts") or you can combine it with languages (though make sure not to count a language with two dialects 3 times...): up to you. Does that make sense?